tests/Output/StreamOutputTest.php

```diff
@@ -54,6 +55,20 @@ public function testReadingLineReadsFromInputStream(): void
         $this->assertEquals('foo', $this->output->readLine());
     }
 
+    public function testReadingLineThrowsRuntimeException(): void
+    {
+        $inputStream = fopen(__DIR__ . '/../../composer.json', 'r');
```

Comment:
For consistency with other tests that interact with files, can you add a `tmp` directory to the same directory as this unit test with a file inside called `output.txt`?  Then, add a couple lines of fake data to it, and use `__DIR__ . '/tmp/output.txt` as the `$inputStream` and `$outputStream`.